evap/staff/templates/quick-review.js

```diff
@@ -111,6 +113,37 @@ $(document).ready(() => {
         }
     }
 
+    let nextEvaluationIndex = 0;
+    updateNextEvaluation(0);
+
+    $("[data-skip-evaluation]").on("click", event => {
+        const skippedEvaluationId = $("[data-evaluation]").not(":hidden").data("evaluation");
+        if (skippedEvaluationId !== undefined) {
```

Comment:
In what case is this equal to undefined?

Reply:
When a user presses the "skip" key, the button is pressed (virtually). This check assures that no AJAX request is sent if the "skip evaluation" button is hidden, in which case skippedEvaluationId is undefined